### PR TITLE
use mkdocs instead

### PR DESCRIPTION
**Pull request by dholbach**
https://test-ignite.readthedocs.io/en/latest/ is what the docs would look like (no webhook to build docs automatically set up).

### CODE REVIEW

**Comment by dholbach** [comment]
@luxas @chanwit @stealthybox I'd appreciate a look over https://test-ignite.readthedocs.io/ and this PR.

Things that are hard to do with `mkdocs` are automatic toctrees. I even attempted to use jinja2 macros, but didn't get anywhere. Everything else becomes easier. The navbar on the the left is still tweakable - right now it's auto-generated.

**Comment by dholbach** [comment]
Not sure what https://circleci.com/gh/weaveworks/ignite/1115 is about. I might need help to fix it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ for binary in ignite ignited; do
 done
 ```
 
-A more throughout installation guide is available here: https://ignite.readthedocs.io/en/latest/installation.html
+A more throughout installation guide is available here: https://ignite.readthedocs.io/en/latest/installation
```
Comment (stealthybox) [inline CHANGELOG.md:179]: Is there a way to redirect the old link?
It's present in every GitHub release currently:
https://github.com/weaveworks/ignite/releases

Comment (dholbach) [reply CHANGELOG.md:179]: I don't quite know how to do that. Can we change the release announcements to point to /en/<release>/rest-of-link.html?

**Comment by dholbach** [comment]
Shall I split up the PR into 1) move to mkdocs and 2) add link-checking?

**Comment by dholbach** [comment]
Shall I split the `mkdocs` move part of the PR out and move the link-checking bits to a separate PR?

**Comment by dholbach** [comment]
Moving the checking of links to https://github.com/weaveworks/ignite/pull/573

**Pull request merged**